### PR TITLE
docs: Fix typos

### PR DESCRIPTION
**Pull request by deining**
This PR fixes a few typos I spotted in the project.

### CODE REVIEW

File: .github/workflows/linter.yml [hunk 1]
```diff
@@ -54,7 +54,7 @@ jobs:
       # Run Linter against code base #
       ################################
       - name: Lint Code Base
-        uses: super-linter/super-linter@v6.3.0
+        uses: super-linter/super-linter@v6.5.1
```
Comment (powersj) [inline .github/workflows/linter.yml:57]: Looking at master it looks like this is already done: https://github.com/influxdata/telegraf/blob/master/.github/workflows/linter.yml#L57

File: .github/workflows/readme-linter.yml [hunk 2]
```diff
@@ -11,7 +11,7 @@ jobs:
     steps:
       - uses: actions/setup-go@v5
         with:
-          go-version: '1.22.1'
+          go-version: '1.22.3'
```
Comment (powersj) [inline .github/workflows/readme-linter.yml:14]: This is already done as well

**Review by powersj — APPROVED** [review]
Thanks for the PR, can you rebase on master please?

**Comment by deining** [comment]
> Thanks for the PR, can you rebase on master please?

Done!

File: .github/workflows/linter.yml (hunk 1 above)
Comment (deining) [reply .github/workflows/linter.yml]: You are right here! No action needed here.

File: .github/workflows/readme-linter.yml (hunk 2 above)
Comment (deining) [reply .github/workflows/readme-linter.yml]: You are right here! No action needed here.

**Review by powersj — APPROVED** [review]

**Review by srebhan — APPROVED** [review]
Thanks for correcting those @deining!

**Pull request merged**